third_party/move/move-binary-format/src/file_format.rs

```diff
@@ -850,28 +839,23 @@ impl AccessSpecifier {
 )]
 pub enum AccessKind {
     Reads,
-    Writes,
-    Acquires, // reads or writes
+    Writes, // Includes reading
```

Comment:
Maybe expand the comment saying that for all writes in Move, they require a read-before-write property or something along these lines